apps/google-analytics-4/frontend/src/components/config-screen/map-account-property/MapAccountPropertySection.tsx

```diff
@@ -58,10 +63,15 @@ export default function MapAccountPropertySection(props: Props) {
   };
 
   return (
-    <Stack spacing="spacingL" marginBottom="none" flexDirection="column" alignItems="flex-start">
+    <Stack spacing="spacingXs" marginBottom="none" flexDirection="column" alignItems="flex-start">
+      <Box>
+        <Subheading>Assign account and property</Subheading>
```

Comment:
> Select account and property

is my suggestion here. That differentiates it from "Assign content types" below.